src/video/rpi_cam.cc

```diff
@@ -40,7 +40,7 @@ RPiCamera::getOutputSize() const
 bool
 RPiCamera::readFrame(cv::Mat &outFrame)
 {
-    uint8_t buffer[72 * 6 * 3];
+    char buffer[72 * 6 * 3];
```

Comment:
Is there no ``uint8_t`` in Windows? I thought it lived in ``cstdlib`` -- not that it matters here.